src/app/shared/brapi/2.0/model/germplasm.ts

```diff
@@ -22,7 +22,9 @@ export interface Germplasm {
   genus?: string;
   germplasmDbId?: string;
   germplasmName?: string;
-  germplasmOrigin?: Array<GermplasmGermplasmOrigin>;
+  // FIXME array on pedigree sync branch?
+  // germplasmOrigin?: Array<GermplasmGermplasmOrigin>;
+  germplasmOrigin?: GermplasmGermplasmOrigin;
```

Comment:
it seems it's an array on [the specs](https://github.com/plantbreeding/BrAPI/blob/V2.0/Specification/BrAPI-Germplasm/Germplasm/Schemas/GermplasmNewRequest.yaml#L285), I kept the non-array version when merging from master just in case.
TODO verify and fix

Reply:
https://ibplatform.atlassian.net/browse/IBP-5725?focusedCommentId=55020
https://ibplatform.atlassian.net/browse/IBP-5989